diesel/src/sqlite/connection/functions.rs

```diff
@@ -70,4 +70,12 @@ impl<'a> Row<Sqlite> for FunctionRow<'a> {
             .iter()
             .all(|&p| unsafe { SqliteValue::new(p) }.is_none())
     }
+
+    fn column_count(&self) -> usize {
+        self.args.len()
+    }
+
+    fn column_name(&self) -> &str {
+        unimplemented!()
```

Comment:
@sgrif I'm not sure how to implement this here. Or if it should even be implemented …

Reply:
You're reaching the fundamental problem with how folks think of SQL vs how it actually works. A field in a select clause does not have a name by definition. It may coincidentally have one, but fundamentally the return value of a `SELECT` statement is a tuple, not a struct. I'm really not sure that there's a good answer here, but this should probably at minimum return `Option<&str>`.

Reply:
I've changed that to `Option<&str>`